telepresence/proxy/deployment.py

```diff
@@ -118,7 +118,7 @@ def remove_existing_deployment(quiet=False):
     command = [
         "run",  # This will result in using Deployment:
         "--restart=Always",
-        "--limits=cpu=100m,memory=256Mi",
+        "--limits=1000m,memory=256Mi",
```

Comment:
I'm not sure how this is interpreted. Can you try `--limits=cpu=1000m,memory=256Mi` (i.e. add back the `cpu=` bit) on your setup?

Reply:
oops :P. will re do my tests with correct setting

Reply:
Tested both ```cpu=1000m``` and without cpu limit, almost getting same performance. 

Also updated PR